decorators/1.py

```diff
@@ -0,0 +1,18 @@
+from functools import wraps
+
+def logged(func):
+    @wraps(func)
+    def inner(*args, **kwargs):
+        saved_args = locals()['args']
+        saved_kwargs = locals()['kwargs']
+        func_result = func(*args, **kwargs)
+
+        print(f"""You called function named: '{func.__name__}',\nwith this args: '{saved_args}',\nand kwargs: '{saved_kwargs}'.\nThis function returns: {func_result}""")
```

Comment:
czy w ten sposób nie działało by tak samo

```suggestion
        func_result = func(*args, **kwargs)

        print(f"""You called function named: '{func.__name__}',\nwith this args: '{args}',\nand kwargs: '{kwargs}'.\nThis function returns: {func_result}""")
```